src/components/main-content.tsx

```diff
@@ -0,0 +1,37 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./base";
+
+import "./main-content.sass";
+import { InvestigateDialogComponent } from "./investigate-dialog";
+import { InvestigationPanelComponent } from "./investigation-panel";
+
+interface IProps extends IBaseProps {}
+interface IState {}
+
+@inject("stores")
+@observer
+export class MainContentComponent extends BaseComponent<IProps, IState> {
+
+  public render() {
+    return (
+      <div className="main-content">
+        {this.renderDialog()}
+      </div>
+    );
+  }
+
+  private renderDialog() {
+    const {ui} = this.stores;
+    const {showInvestigationModalSelect} = this.stores.ui;
+    const {showInvestigationPanel} = this.stores.ui;
+    // {return ui.showInvestigationModalSelect ? "true" : "false"}
+    if (showInvestigationModalSelect) {
+      return <InvestigateDialogComponent/>;
```

Comment:
Do we really want the modal to hide the active investigation? I'd think it would just darken it like in the mockup.

Reply:
This entire chunk needs to be reworked to handle multiple investigation types and we need to add the semi-opaque curtain behind the dialog.  This might be best done in another story (possibly the same as the above comment).